pages/common/cowsay.md

```diff
@@ -0,0 +1,15 @@
+# cowsay
+
+> Generate an ASCII character like a cow or sheep saying or thinking something
+
+- Print an ASCII cow saying "Hello world!"
+
+`cowsay {{"Hello world!"}}`
+
+- Print an ASCII dragon telling you what date it is
+
+`date | cowsay -f {{dragon}}`
```

Comment:
We also agreed that we try to avoid piping other commands except the most basic ones (`echo` and `ls`). So in your case it should be

```md
`echo "Hello" | cowsay -f dragon`
```